### PR TITLE
apisonator: release 2.87.1

### PR DESCRIPTION
**Pull request by miguelsorianod**

### CODE REVIEW

**Comment by davidor** [comment]
We need to update `Gemfile.on_prem.lock` too.

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,13 @@
 
 Notable changes to Apisonator will be tracked in this document.
 
+## 2.87.1 - 2018-06-27
+
+#### Fixes
```
Comment (davidor) [inline CHANGELOG.md]: There's an extra '#'.

Comment (miguelsorianod) [reply CHANGELOG.md]: Done :+1:

**Pull request merged**